src/runtime/databases/node/http.ts

```diff
@@ -30,10 +30,10 @@ export class HttpDatabase extends Database {
       options.body = request.body[0];
     }
     httpRequest(options, (error, response, body) => {
-      // console.log("GOT RESPONSE", response.statusCode);
-      // console.log(error);
-      // console.log(response);
-      // console.log(body);
+      if(error || !response) {
+        // @TODO: expose errors and other weirdness into Eve instead of just bailing here.
```

Comment:
Is there any reason not to stick a `console.warning(...)` here to indicate the failure in the interim?